app/models/concerns/api_action_caching.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# API controllers inherit from ActionController::Metal to keep them slim and fast.
+# This concern adds the minimum requirements needed to use Action Caching in the API.
+
```

Comment:
This is exciting! Normally in an API endpoint we go through:

1. Fetching a bunch of data
2. Serializing it
3. Rendering the output

All 3 of these steps are CPU-bound. By using Action Caching in controllers (similar to Fragment Caching in views) we can cache the entire output of an action, _after_ the render. So we're caching after point 3, instead of after point 1 or 2.

Reply:
Yes, it'll be a huge win if we can use action caching. My biggest concern are query params since action caching doesn't seem to use them in the cache key but there seems to be an option for that: https://stackoverflow.com/a/2166419/2838228

Reply:
I did some testing and it seemed to use the entire URL as the cache key, which included order id and distributor id in this case. I'll take a closer look...

Reply:
Updated :+1: